- description: Inserting a bigint into an int should cause an error, see #576.
  url: /v1/graphql
  response:
    errors:
    - extensions:
        path: $.selectionSet.insert_test_types.args.objects[0].c2_integer
        code: parse-failed
      message: The value 2.147483648e9 lies outside the bounds or is not an
               integer.  Maybe it is a float, or is there integer overflow?
  status: 200
  query:
   query: |
     mutation {
       insert_test_types(
         objects: [
           {
           c2_integer : 2147483648
           }
         ]
       ) {
         returning {
           c2_integer
         }
       }
     }
